earthvision/models/coatnet.py

```diff
@@ -0,0 +1,265 @@
+"""Inspired by https://github.com/chinhsuanwu/coatnet-pytorch"""
+import torch
+import torch.nn as nn
+from einops import rearrange
```

Comment:
add library ke requirements.txt and environment.yml @dewabratapandu

Reply:
done